docs/v3/advanced/infrastructure-as-code.mdx

```diff
@@ -22,6 +22,195 @@ to reach [parity with the Prefect API](https://github.com/PrefectHQ/terraform-pr
+        
+        ```bash
+        pulumi new python \
+        --yes \
+        --generate-only \
+        --name "my-prefect-pulumi-project" \
+        --description "A Pulumi project to manage Prefect resources" \
+        --runtime-options toolchain=uv \
+        --runtime-options virtualenv=.venv
+        ```
+        
+        <Warning>
+            Don't name your project any of the following, otherwise you will have package name conflicts:
+            * `pulumi`
+            * `prefect`
+            * `pulumi-prefect`
+        </Warning>
+        
+        <Info>
+            An explanation of the [flags](https://www.pulumi.com/docs/iac/cli/commands/pulumi_new/#options) used:
+            * The `--yes` flag skips the interactive prompts and accepts the defaults. You can omit this flag, 
+            or edit the generated `Pulumi.yaml` file later to customize your project settings.
+            * The `--generate-only` just creates a new Pulumi project. It does not create a stack, save config, 
+            or install dependencies.
+            * The `--name` and `--description` flags set the name and description of your Pulumi project.
+            * The `--runtime-options toolchain=uv` and `--runtime-options virtualenv=.venv` flags configures 
+            the Pulumi project to use the `uv` toolchain instead of the default, `pip`.
+        </Info>
+        
+        To finish setting up your new Pulumi project, navigate to the project directory and install the dependencies:
+        
+        ```bash
+        pulumi install
+        ```
+    </Accordion>
+    <Accordion title="Using an existing Pulumi project">
+        If you already have a Pulumi project, you can switch to the `uv` toolchain by updating 
+        your `pulumi.yaml` file `runtime` settings as shown below:
+
+        ```yaml Pulumi.yaml
+        # other project settings...
+        runtime:
+          name: python
+          options:
+            toolchain: uv
+            virtualenv: .venv
+        ```
+        
+        This configures your Pulumi project to use the `uv` toolchain and the virtual environment located at `.venv`.
+        
+        Run the following to update Pulumi to use the `uv` toolchain:
+
+        ```bash
+        pulumi install
+        ```
+    </Accordion>
+</AccordionGroup>
+
+### Managing Resources with Pulumi
+
+To manage resources with Pulumi, add the Prefect Terraform provider to your Pulumi project:
+
+```bash
+pulumi package add terraform-provider prefecthq/prefect
+```
+
+Optionally, you can specify a specific version, e.g.:
+```bash
+pulumi package add terraform-provider prefecthq/prefect 2.90.0
+```
+
+This will auto-generate a `pulumi-prefect` Python package. 
+The code will be placed in the `sdks/prefect` directory inside the Pulumi project.
+
+### Example: Deploying a Flow with Pulumi
+
+This simple example shows you how to deploy a flow to Prefect.
+
+```python
```

Comment:
otherwise the markdown tests will try to run this as is
```suggestion
{/* pmd-metadata: notest */}
```python
```

Reply:
Thanks for the pointer! Fixed.